whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisSearchServiceTestIntegration.java

```diff
@@ -2083,21 +2083,16 @@ public void search_route6_roa_mismatch_less_specific_as_xml_strings() {
                 "        <attribute name=\"source\" value=\"TEST-NONAUTH\"/>\n" +
                 "    </attributes>\n" +
                 "    <objectmessages>\n" +
-                "        <objectmessage severity=\"Warning\" text=\"Warning: this %s object conflicts with an " +
-                "overlapping RPKI ROA with a less specific prefix %s but same origin AS%s.&#xA;As a result an announcement for this prefix may be" +
-                " rejected by many autonomous systems. You should either remove this route: object or update or delete the ROA.&#xA;\">\n" +
-                "            <args value=\"route6\"/>\n" +
+                "        <objectmessage severity=\"Warning\" text=\"Warning: this %s object conflicts with an RPKI ROA with a less specific prefix /%s but same origin AS%s.&#xA;As a result, many autonomous systems may reject a BGP announcement even if it matches the ROUTE object. You should consider either removing this ROUTE object or updating/deleting the RPKI ROA.&#xA;\">\n" +
+                "            <args value=\"ROUTE6\"/>\n" +
                 "            <args value=\"32\"/>\n" +
                 "            <args value=\"52511\"/>\n" +
                 "        </objectmessage>\n" +
                 "    </objectmessages>\n" +
                 "</object>\n" +
                 "</objects>\n" +
                 "<terms-and-conditions xlink:type=\"locator\" xlink:href=\"https://apps.db.ripe.net/docs/HTML-Terms-And-Conditions\"/>\n" +
-                "<version " +
-                "version=\"" + applicationVersion.getVersion() + "\" " +
-                "timestamp=\"" + applicationVersion.getTimestamp() + "\" " +
-                "commit-id=\"" + applicationVersion.getCommitId() + "\"/>\n" +
+                "<version version=\"0.1-TEST\" timestamp=\"20240610-1352\" commit-id=\"0\"/>\n" +
```

Comment:
Why are you hard-coding the variables? The `getTimestamp()` will change and fail the test. Use the `applicationVersion` instead (as before)